src/components/details-view/details-view.js

```diff
@@ -272,7 +274,14 @@ export default class DetailsView extends Component {
               data-test-eholdings-details-view-list={type}
             >
               <div className={styles['list-header']}>
-                <h3>{capitalize(listType)}</h3>
+                <div>
+                  <h3>{capitalize(listType)}</h3>
+                  {resultsLength && (
+                    <div data-test-eholdings-details-view-results-count>
+                      <p><small>{resultsLength} records found</small></p>
```

Comment:
I realized we'll have to deal with the singular version here if only 1 is found, but we can deal with that when we start to look at localizing strings.